packages/direflow-component/src/config/config-overrides.ts

```diff
@@ -5,14 +5,21 @@ import fs from 'fs';
 import { resolve } from 'path';
 import { PromiseTask } from 'event-hooks-webpack-plugin/lib/tasks';
 
-module.exports = function override(config: any, env: string): any {
+interface IOptions {
+  filename?: string;
+  chunkFilename?: string;
+}
+
+module.exports = function override(config: any, env: string, options?: IOptions): any {
+  const { filename = 'direflowBundle.js', chunkFilename = 'vendor.js' } = options || {};
```

Comment:
I think this line may be a bit confusing.
To make the intention more clear, I would prefer the more verbose version:
```typescript
const filename = options?.filename || 'direflowBundle.js';
const chuckFileName = options?.chunkFilename || 'vendor.js';
```
What do you think? 🙂

Reply:
sure if you prefer that

Reply:
changes have been pushed